cypress/e2e/smoke/load_safe.cy.js

```diff
@@ -19,8 +19,8 @@ describe('Load existing Safe', () => {
     cy.visit('/welcome?chain=matic')
     cy.contains('Accept selection').click()
 
-    // Enters Loading Safe form
-    cy.contains('Add existing Safe').click()
+    // Enters Loading Safe form - force if banner is present
+    cy.contains('Add existing Safe').click({ force: true })
```

Comment:
Wouldn't it be better to close the banner in `before()`?

Reply:
We'd need to remove it with the banner then, and dynamic Beamer banners would also break the test.